unfoldingWord/translationCore
# Fixed online import & added loading circle for user experience

**PLEASE REVIEW THIS PR AFTER MY OTHER PR IS MERGED WITH DEVELOP**

#### This pull request addresses:

- Fixed online import 
- added loading circle for user experience   

#### How to test this pull request:
1. open your console then go to the network tab.
2. click where it says **No throttling ** and select GPRS to slow down your download bandwidth.
3. open the modal under the **import online project** tab 
4. type in a git repo. ex. https://git.door43.org/romastandarddemo/rmy-x-vg_luk_text_ulb.git
5. then click **Import**.

a loading circle should show up and then once downloaded open the tools tab.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1360)
